web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2191,10 +2199,13 @@ public VoidResponse adminCancel(@Required("id") @PathParam("id") String id) {
                     query.cancel();
                     close(query);
                 } catch (Exception e) {
-                    // if this is a query that is in the old query cache, then we have already successfully closed this query so ignore
-                    if (oldQueryCache.getIfPresent(id) == null) {
+                    log.warn("Failed to adminCancel " + id + ", checking if closed previously");
```

Comment:
Same as for adminClose... I'm not sure we want to "hide" the error if the admin cancels a query that has already completed.